src/V3Dfg.h

```diff
@@ -331,9 +331,11 @@ class DfgVertex VL_NOT_FINAL {
         // For simplicity, all packed types are represented with a fixed type
         if (AstUnpackArrayDType* const typep = VN_CAST(nodep->dtypep(), UnpackArrayDType)) {
             // TODO: these need interning via AstTypeTable otherwise they leak
-            return new AstUnpackArrayDType{typep->fileline(),
-                                           dtypeForWidth(typep->subDTypep()->width()),
-                                           typep->rangep()->cloneTree(false)};
+            AstNodeDType* const dtypep = new AstUnpackArrayDType{
```

Comment:
Remove TODO comment above.  I don't know why we didn't put it into the type table earlier.